.github/workflows/tests.yml

```diff
@@ -29,30 +29,30 @@ jobs:
       options: --user 1001
     services:
       meilisearch:
-        image: getmeili/meilisearch:v0.28.0
+        image: getmeili/meilisearch:v1.0
```

Comment:
Shouldn't the image be [v1.0.1](https://hub.docker.com/layers/getmeili/meilisearch/v1.0.1/images/sha256-22a9ab1db3b33e676305a3d6c5d43ec20a294a1a3f023f3130ab77aaa14cfc95?context=explore)?